jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/AppServerAutoDetectionTest.java

```diff
@@ -121,18 +123,18 @@ public void testDefaultServer() throws IOException {
             .build()).build();
 
         AppServerHandler appServerHandler = new AppServerDetector(generatorContext).detect(null);
-        assertEquals("tomcat", appServerHandler.getName());
+        assertThat(appServerHandler.getName()).isEqualTo("tomcat");
     }
 
     private void assertAppServerDescriptorApplicability(Object[] descriptors) throws IOException {
         for (int i = 0; i < descriptors.length; i += 2) {
             String descriptor = (String) descriptors[i];
             boolean expected = (boolean) descriptors[i + 1];
 
-            File appDir = folder.newFolder("webapp" + i);
-            assertTrue(new File(appDir, "META-INF/").mkdirs());
-            assertTrue(new File(appDir, "WEB-INF/").mkdirs());
-            assertTrue(new File(appDir, descriptor).createNewFile());
+            File appDir = Files.createDirectory(folder.resolve("webapp" + i)).toFile();
+            new File(appDir, "META-INF").mkdirs();
+            new File(appDir, "WEB-INF").mkdirs();
+            new File(appDir, descriptor).createNewFile();
```

Comment:
I guess my comment was not clear. Please use nio Files utility methods that don't return boolean false in case of failure but  throw an exception (Runtime) instead.
```suggestion
            Path webInf = Files.createDirectories(folder.resolve("webapp" + i)
                .resolve("META-INF").resolve("WEB-INF"));
            Files.createFile(webInf.resolve(descriptor));
```